new-dti-website/components/course/data/testimonials.json

```diff
@@ -1,39 +1,22 @@
 {
   "testimonials": [
     {
-      "description": "It was so great yeah you should take it like it was crazy like all the best things in the world would want to go take the class like I feel enlightened",
-      "name": "NAME/ANONYMOUS 1",
-      "semesterTaken": "Fall 2023"
+      "description": "Trends in Web Development has been an incredibly valuable course, equipping me with practical skills and knowledge that will greatly benefit my future career. The final project was a rewarding experience, allowing me to put my new skills into practice and create a project I'm proud of.",
+      "name": "Juju Crane",
+      "semesterTaken": "Fall 2024",
+      "profileImage": "/course/testimonials/jlc565.jpg"
     },
     {
-      "description": "It was so great yeah you should take it like it was crazy like all the best things in the world would want to go take the class like I feel enlightened",
-      "name": "NAME/ANONYMOUS 1",
-      "semesterTaken": "Fall 2023"
+      "description": "Trends was a great class because of the hands-on webdev practice you get. It helped me see web development as a whole, and also gave me a project that I could use in my portfolio.",
```

Comment:
Would it be bad to include exclamation points in these? I know it might be twisting their own descriptions since they are testimonials, but it could make them seem a bit more exciting.

Reply:
I think it wouldn't hurt. I actually edited some exclamation points out by accident when cutting these short.